src/components/Timer/Timer.tsx

```diff
@@ -1,5 +1,66 @@
 import React from 'react';
+import { makeStyles } from '@material-ui/core';
 
-const Timer: React.FC = () => <h1>Timer</h1>;
+const useStyles = makeStyles(() => ({
+  timer: {
+    position: 'fixed',
+    left: 0,
+    width: '100vw',
+    height: '100vh',
+    margin: 0,
+    textAlign: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+  },
+}));
+
+const Timer = () => {
+  const classes = useStyles();
+  const [minutes, setMintues] = React.useState(0);
+  const [seconds, setSeconds] = React.useState(-5);
```

Comment:
I like prefixing state variables with `s` to quickly identify them when reading through the code.